### PR TITLE
Fix issue related to `save-all` path

### PR DESCRIPTION
**Pull request by dennisschagt**
Fixes https://github.com/newsboat/newsboat/issues/1418

### CODE REVIEW

File: src/dirbrowserformaction.cpp [hunk 1]
```diff
@@ -295,24 +284,20 @@ void DirBrowserFormAction::init()
...
-
-		dir = save_path;
-	}
+	LOG(Level::DEBUG,
+		"view::dirbrowser: save-path is '%s'",
+		save_path);
 
-	int status = ::chdir(dir.c_str());
-	LOG(Level::DEBUG, "view::dirbrowser: chdir(%s) = %i", dir, status);
+	int status = ::chdir(save_path.c_str());
```
Comment (der-lyse) [inline src/dirbrowserformaction.cpp:293]: I reckon this can now be `const` as well.

**Review by der-lyse — APPROVED** [review]
Fixing bugs by removing code are the very best fixes! :-)

I confirm, this fixes the bug.

File: src/dirbrowserformaction.cpp (hunk 1 above)
Comment (dennisschagt) [reply src/dirbrowserformaction.cpp]: Thanks, suggestion squashed into https://github.com/newsboat/newsboat/pull/1421/commits/0f31b9da5dd8ded7a7178c1986867ceb5af802b1.

**Review by Minoru — APPROVED** [review]
Thanks! This looks excellent to me. :+1:

The first two commit messages were a bit confusing because at the time, `dir` *was* used in both classes. But the last two commits made it all make sense in the end :)

Anyone can merge this once the CI is done and Lyse gave his approval.

**Review by der-lyse — APPROVED** [review]
Reapproved! :-)

**Comment by dennisschagt** [comment]
Thank you for the reviews!

**Pull request merged**